lib/modules/manager/pep621/extract.spec.ts

```diff
@@ -134,14 +134,14 @@ describe('modules/manager/pep621/extract', () => {
           datasource: 'pypi',
           depType: 'project.optional-dependencies',
           currentValue: '>12',
-          depName: 'pytest/pytest',
```

Comment:
These are changes from a commit for pep621 grouping changes

Reply:
I think these were rebase-related and now resolved